src/app/payment/paymentMethod/HostedWidgetPaymentMethod.tsx

```diff
@@ -226,6 +228,8 @@ class HostedWidgetPaymentMethod extends Component<
                         method={ method }
                         onSignOut={ this.handleSignOut }
                     /> }
+
+                    { renderAdditionalElements && renderAdditionalElements() }
```

Comment:
This is still outside the designated container. Almost identical to the last solution. If the problem is that `renderContainer()` fails to render the element you want, we should aim to improve the way of invoking `renderContainer()`.

Reply:
The div in renderContainer sets the display style to none when certain conditions are met, which deletes the custom payment form; this behavior is expected in payment methods that use custom payment forms.
Here I am trying to render some elements to load a stylesheet and get the styles from it, and I want it to always be rendered, so I shouldn't use the same renderCustomPaymentForm prop that is used for different purposes